wrappers/swift/Logger.swift

```diff
@@ -0,0 +1,226 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper class around ObjC Logger class `ODWLogger` used to events.
+public class Logger {
```

Comment:
Do we expect subclasses of this class? If not, we should mark this as `final` to prevent unexpected behaviors if someone chooses to subclass.

Reply:
Don't see it now but in future it might get subclassed as it has lots of instance methods. So keeping it non-final.

Reply:
The problem is that these classes don't seem designed for subclassing since they are modifying class properties instead of instance properties. The state management can get pretty complicated.

Reply:
Not sure if I understand the comment. Can you please explain the class properties vs instance properties? As I see this class is indeed being instantiated and modifying via its instance methods.

Reply:
While this class is modifying its internal state via instance variables, there isn't a defined contract on how other classes should subclass this. As a subclasser, will I be required to call super at some point in my implementation? Do I do that at the start or at the end of my implementation? If I override on function, do I need to override the others?